packages/vaadin-login/src/vaadin-login-form.js

```diff
@@ -114,6 +114,9 @@ class LoginFormElement extends LoginMixin(ElementMixin(ThemableMixin(PolymerElem
   connectedCallback() {
     super.connectedCallback();
     this._handleInputKeydown = this._handleInputKeydown.bind(this);
+    if (this.autoFocusUsername) {
+      this.$.vaadinLoginUsername.focus();
+    }
```

Comment:
Should it use the TextField's `autofocus` feature instead?

Reply:
That's an option worth trying but in theory it might not work if timing is not correct.